lua/lspconfig/configs/lua_ls.lua

```diff
@@ -8,23 +8,14 @@ local root_files = {
   'stylua.toml',
   'selene.toml',
   'selene.yml',
+  '.git',
 }
 
 return {
   default_config = {
     cmd = { 'lua-language-server' },
     filetypes = { 'lua' },
-    root_dir = function(fname)
-      local root_file = util.root_pattern(unpack(root_files))(fname) or ''
-      local root_lua = util.root_pattern 'lua/'(fname) or ''
-      local root_git = vim.fs.dirname(vim.fs.find('.git', { path = fname, upward = true })[1]) or ''
-      if root_file == '' and root_lua == '' and root_git == '' then
-        return
-      end
-      local root = #root_file >= #root_lua and root_file or root_lua
```

Comment:
Yeah this is wrong.

If a project has a `.luarc.json` and a `lua/`, the latter would get chosen, which is wrong. `.luarc.json` should always win.

Reply:
To be clear, are you +1 on this PR? :)